util/metrics/counter.go

```diff
@@ -66,13 +66,12 @@ func (counter *Counter) Inc(labels map[string]string) {
 	if len(labels) == 0 {
 		counter.fastAddUint64(1)
 	} else {
-		counter.Add(1.0, labels)
+		counter.addLabels(1.0, labels)
 	}
 }
 
-// Add increases counter by x
-// For adding an integer, see AddUint64(x)
-func (counter *Counter) Add(x float64, labels map[string]string) {
+// addLabels increases counter by x
+func (counter *Counter) addLabels(x uint64, labels map[string]string) {
```

Comment:
I like that this was changed and nothing was lost because everything was already using AddUint64